terraform/cloudfoundry/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "zones" {
+  description = "AWS availability zones"
+  default     = {
+    zone0 = "eu-west-1a"
+    zone1 = "eu-west-1b"
+    zone2 = "eu-west-1c"
+  }
+}
+
+variable "cf_cidrs" {
+  description = "CIDR for cf components subnet indexed by AZ"
+  default     = {
+    zone0 = "10.0.10.0/24"
+    zone1 = "10.0.11.0/24"
+    zone2 = "10.0.12.0/24"
+  }
+}
+
+variable "dns_zone_id" {
+  description = "Amazon Route53 DNS zone identifier"
+  default = "Z3SI0PSH6KKVH4"
+}
+
+variable "dns_zone_name" {
+  description = "Amazon Route53 DNS zone name"
+  default     = "cf.paas.alphagov.co.uk"
+}
+
+variable "vpc_id" {
+  description = "id of VPC created in main 'vpc' terraform"
+}
+
+variable "AWS_ACCESS_KEY_ID" {
+  description = "AWS access key to be pass to the bosh CPI"
```

Comment:
Despite [this comment](https://github.com/alphagov/paas-cf/pull/28#issuecomment-170898565) this description is wrong. The AWS credentials in this case are added to be passed to the cloudcontroller to be able to access the S3 buckets.